apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -869,9 +877,15 @@ export default function EnrollForm(props: EnrollFormProps) {
         Regional Partners and facilitators are contractually obliged to treat
         this information with the same level of confidentiality as Code.org.
       </p>
-      <Button id="submit" onClick={handleClickRegister} disabled={isSubmitting}>
-        Register
-      </Button>
+      <Button
+        id="submit"
+        isIconOnly={false}
+        text={'Register'}
+        color={buttonColors.purple}
+        style={{color: 'white'}}
```

Comment:
Shouldn't the default have white text? Do you know why that isn't working correctly?

Reply:
That's what I thought too but it was showing black, this file doesn't import styles though so that's why I just hard-coded it here since other components were hard-coding styles.

Reply:
I'm in the middle of refactoring this form to use dsco components, and I found bootstrap imported in its stylesheet. I'm guessing that's the culprit. removing it in my branch, so once this code merges I can clean it up

![Screenshot 2024-11-18 at 2 01 13 PM](https://github.com/user-attachments/assets/6f7adc63-b0c5-4db2-8b34-8a4119cfcc7b)

Reply:
Cool, so Ryan will clean this up? That SGTM!

I know this is a nit, but I'd rather not keep unnecessary inline styling around!

Reply:
I could wait for Ryan's fix then merge it in without the inline change? Otherwise I think we'll have to decide between some temporarily inline styling or temporarily having the button text be black?

Reply:
oh, another option might be to swap in the dsco buttons right away? my pr is turning out to be a pretty drastic refactor, so I think it would be easier if I handle the conflicts 😬

Reply:
fwiw, temporary inline styling is completely fine to me. Do whatever is easier for you both :)